viritin/src/main/java/org/vaadin/viritin/form/AbstractForm.java

```diff
@@ -130,10 +130,18 @@ public void setEntity(T entity) {
     /**
      * @return true if bean has been changed since last setEntity call.
      */
-    public boolean hasChanges() {
+    protected boolean hasChanges() {
         return hasChanges;
     }
 
+    protected void setHasChanges(boolean hasChanges) {
+        this.hasChanges = hasChanges;
+    }
+
+    protected boolean isValid() {
```

Comment:
I guess this could be public as well? Might be handy in certain places, so one don't need to get the binder separatly.

Reply:
This could be public. I will update the PR.